platforms-serverless/netlify-ci/README.md

```diff
@@ -7,7 +7,7 @@ Prisma and Netlify integration via Git (via another separate Git repository that
 
 ### Environment variables
 
-The environment variable `NETLIFY_PG_URL` should point to a postgres database.
+The environment variable `NETLIFY_BETA_PG_URL` should point to a postgres database.
```

Comment:
nit: It's not in beta anymore I think?

Reply:
True, we should rework all the secrets in the future though so not trying to do that right now (as it also requires to update it on Netlify...).